governance/votes/Executive vote - February 22, 2024.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Spark Proxy Spell - February 22, 2024
+summary: Stability Fee changes for ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C, Spark Protocol DC-IAM parameter changes, pushing USDP out of input conduit, yanking BA Labs DAI and MKR vest streams, triggering Spark proxy spell.  
+date: 2024-02-22T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Increasing wstETH Supply Cap on Spark Ethereum, Increasing the Spark DAI Effective Borrow APY, Pushing USDP out of Input Conduit, Yanking Vest Streams, Triggering Spark Proxy Spell - February 22, 2024
```

Comment:
This should match the title. Please, check the Executive Copy template: https://github.com/makerdao/community/blob/master/governance/votes/templates/Executive%20vote%20-%20%24date_MONTH_DD%2C_YYYY.md

```suggestion
# [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Spark Proxy Spell - February 22, 2024
```